index.html

```diff
@@ -142,152 +153,14 @@ <h4>Cultural Transformation</h4>
         <div class="button-box">
           <button class="btn btn-main">Join DevOps summit 2023</button>
         </div>
-        <div class="full-program"><a class="full-a" href="#">See The Full Program</a></div>
+        <div class="full-program">
+          <a class="full-a" href="#">See The Full Program</a>
+        </div>
       </section>
       <section class="speakers" id="speakers">
         <h3>Feature Speakers</h3>
         <div class="border"></div>
```

Comment:
- [ ] To align with the design, please make sure to display only two speakers in the mobile version of the `index.html`. Currently, all speakers are being shown, so kindly make the necessary adjustments. **Following the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) will help maintain consistency and enhance the overall visual appeal.** 💪

   ### Your design:
   ![image](https://github.com/alyconr/DevOpsConf2023/assets/107952724/28d7077c-414f-41cb-a529-c269a1129579)

   ### Behance design:
   ![image](https://github.com/alyconr/DevOpsConf2023/assets/107952724/26f2cd18-ac0f-43c0-a6f9-d3bb5816acc1)